cluster-autoscaler/FAQ.md

```diff
@@ -1086,223 +1132,3 @@ If you need to update vendor to an unreleased commit of Kubernetes, you can use
 ```
 ./hack/submodule-k8s.sh <k8s commit sha> git@github.com:kubernetes/kubernetes.git
 ```
-
-Caveats:
```

Comment:
Please revert this entire section
gardener specific docs should be kept

Reply:
done